### PR TITLE
Skip all the modules that should not be deployed to central (separate…

### PR DESCRIPTION
**Pull request by tweise**
… from -Pall-modules).

### CODE REVIEW

**Comment by tweise** [comment]
@davidyan74 @vrozov please merge

File: samples/pom.xml [hunk 1]
```diff
@@ -34,6 +34,11 @@
 
   <name>Apache Apex Malhar (incubating) Samples</name>
 
+  <properties>
+    <semver.plugin.skip>true</semver.plugin.skip>
```
Comment (vrozov) [inline samples/pom.xml]: Is semver plugin enabled for samples? japicmp is part of plugin management and should not be invoked unless explicitly enabled.

Comment (tweise) [inline samples/pom.xml]: I guess not, but this is how I found it in another module. However, why would be require a per module plugin configuration for something that by default should apply to all modules?

Comment (vrozov) [inline samples/pom.xml]: The common to all modules configuration should be part of plugin management and each module may modify it. It is not clear to me why it is necessary to provide semver plugin configuration settings here as semver plugin does not seem to be enabled for the samples module.

Comment (tweise) [inline samples/pom.xml]: No issue with the plugin management part. But I do not see the point of repeating the plugin like this:

```
      <plugin>
        <groupId>com.github.siom79.japicmp</groupId>
        <artifactId>japicmp-maven-plugin</artifactId>
      </plugin>
```

in every module. Instead, I would prefer to see this in the malhar parent pom only and use the property that turns it off for those modules that should be exempt. That's how the deploy plugin is setup already.

Comment (vrozov) [inline samples/pom.xml]: It depends on the percentage of modules that semver should be enabled and disabled for. In Malhar, there are more modules where it needs to be disabled, it seems that repeating 

```
      <plugin>
        <groupId>com.github.siom79.japicmp</groupId>
        <artifactId>japicmp-maven-plugin</artifactId>
      </plugin>
```

only for modules where semver needs to be enabled will be more applicable in this case.

**Comment by tweise** [comment]
I removed the property as it is not related to this PR. I also think we should revisit how we manage semver execution, making it property driven is less verbose and gives more flexibility through command line and profile properties.

**Pull request merged**